jekyll/_cci2/customizing-ios.md

```diff
@@ -0,0 +1,429 @@
+a [config.yml file]( {{ site.baseurl }}/2.0/configuration-reference/) in the `.circleci` directory of your
+repository. 
+* **Install any Ruby gems specified in a Gemfile** - We will run `bundle
+  install` and cache the installed gems.
+* **Install any dependencies managed by CocoaPods** - If a Podfile is
+  present, we will run `pod install`, or `bundle exec pod install` if
+  both a Podfile and a Gemfile are present, and also cache the installed pods.
+* **Run the "test" build action for detected workspace (or project) and scheme
+  from the command line** - We will use `xctool` to build either the workspace
+  we find in your repo, or the project, if there is no workspace.
+
+
+### CocoaPods
+
+CircleCI will automatically detect if your project is using [CocoaPods](https://cocoapods.org)
+to manage dependencies. If you are using CocoaPods, then we recommend that you
+check your [Pods directory into source control](http://guides.cocoapods.org/using/using-cocoapods.html#should-i-check-the-pods-directory-into-source-control).
+This will ensure that you have a deterministic, reproducible build.
+
+If CircleCI finds a `Podfile` and the `Pods` directory is not present (or empty)
+ then we will run `pod install` to install the necessary dependencies in the
+`dependencies` step of your build.
+
+## Supported build and test tools
+
+CircleCI's automatic commands cover a lot of common test patterns, and you can customize your build
+as needed to satisfy almost any iOS build and test strategy.
+
+### XCTest-based tools
+In addition to standard `XCTestCase` tests, CircleCI will automatically run tests
+written in any other tool that builds on top of XCTest and is configured to run
+using the "test" build action. The following test tools are known to work well on CircleCI
+(though many others should work just fine):
+
+* [XCTest](https://developer.apple.com/library/ios/documentation/DeveloperTools/Conceptual/testing_with_xcode/chapters/01-introduction.html)
+* [Kiwi](https://github.com/kiwi-bdd/Kiwi)
+* [KIF](https://github.com/kif-framework/KIF)
+
+### Other tools
+Popular iOS testing tools like [Appium](http://appium.io/) and [Frank](http://www.testingwithfrank.com/) should also
+work normally, though they will need to be installed and called using custom commands.
+
+## Code signing
+[fastlane](https://fastlane.tools) and [Shenzhen](http://nomad-cli.com/#shenzhen)
+are pre-installed on the container image. It is recommended to use fastlane to
+build, sign and distribute your iOS app to beta-testers.
+
+The fastest way to get code signing working on CircleCI is to follow
+these steps:
+
+1. Upload your provisioning profile (`.mobileprovision`) and private key (`.p12`)
+   files in **Project Settings > iOS Code Signing**.
+1. Set `GYM_CODE_SIGNING_IDENTITY` environment variable
+   to match your code-signing identity, ie `"iPhone Distribution: Acme Inc."`.
+1. Build with `gym` and deploy with `ipa`.
+
+## Customizing Your Build
```

Comment:
This section no longer applies as there is no inference in 2.0, and all commands are customised from the start. We should remove this section